dashboard/app/models/script_level.rb

```diff
@@ -143,6 +143,16 @@ def has_another_level_to_go_to?
     end
   end
 
+  def path_for_user
+    build_script_level_path(this_level)
+  end
+
+  # Return this script level
+  def this_level
+    i = script.script_levels.find_index(self)
+    script.script_levels[i]
```

Comment:
why is this necessary? Why can't you use `build_script_level_path(self)` above?

Reply:
Yes - thanks!